README.md

```diff
@@ -99,6 +99,17 @@ The following diagram represents how the information is organised once a certifi
 [![Dgc data organisation diagram](https://github.com/rust-italia/dgc/raw/main/docs/certificate_structure.svg)](https://github.com/rust-italia/dgc/raw/main/docs/certificate_structure.svg)
 
 
+## FAQ
+
+### Is it legal to use this library?
+
+You can certainly use this library to look into your own personal DGC (or the ones provided in the [official test dataset](https://github.com/eu-digital-green-certificates/dgc-testdata)).
+
+If you are trying to use this library to look into certificates of arbitrarty individuals, you need to be aware that you will have access to privacy-sensitive personal information. Privacy-related regulation might limit you or prevent you from using this library (or the data acquired through this library). It is recommended to consult the relevant legal sources and authorities for any significant production use case.
```

Comment:
> Privacy-related regulation

should be _Privacy-related regulation**s**_?

Reply:
> The noun regulation can be countable or uncountable. In more general, commonly used, contexts, the plural form will also be regulation. However, in more specific contexts, the plural form can also be regulations e.g. in reference to various types of regulations or a collection of regulations.

I am confused 😆

Reply:
😅 Ok, I think we can leave it as you wrote it.